docker/windows/Dockerfile

```diff
@@ -20,13 +21,10 @@ LABEL \
 EXPOSE 10933
 
 # Either from a build artifact dependency or local build output.
-COPY _artifacts/msi/Octopus.Tentacle.${BUILD_NUMBER}-x64.msi /tmp/
-
-WORKDIR /tmp
-RUN msiexec /i Octopus.Tentacle.%BUILD_NUMBER%-x64.msi /qn /l*v Tentacle-Installation.log
+COPY _artifacts/tentacle "${INSTALLATION_FOLDER}"
```

Comment:
This is the only way I've found that works with `COPY` and Windows paths that contain spaces, i.e. with an `ARG`. Open to suggestions.